### ISSUE TITLE
Support Vector Regression

### ISSUE THREAD

**Issue by bvpav** [CodenameSource/libuaic#7]
Имплементиране на регресия с помощен вектор(SVR)

# Структура на SVR
- [ ] Структура с коефициенти и указатели към методите на алгоритъма
- [ ] Loss function, фунцкия калкулираща грешката на предсказания, от алгоритъма, резултат от реалния (За едно поле от тренировъчните данни)
- [ ] Optimizer function - Gradient descent, оптимизираща функция
- [ ] Fit function, функция която тренира алгоритъма върху въведените данни
- [ ] Predict function, функция която взима коефициентите и връща предсказания резултат спрямо въведените данни на базата на тренираните коефициенти

# Ресурси

- [Implementing SVR from scratch](https://towardsdatascience.com/implementing-svm-from-scratch-784e4ad0bc6a)
- [Support vector machine from scratch](https://www.pycodemates.com/2022/10/implementing-SVM-from-scratch-in-python.html)
- [Gradient descent](https://philippmuens.com/gradient-descent-from-scratch)
